optuna/cli.py

```diff
@@ -837,7 +837,7 @@ def get_parser(self, prog_name: str) -> ArgumentParser:
         parser.add_argument("--study-name", type=str, help="Name of study.")
         parser.add_argument("--trial-number", type=int, help="Trial number.")
         parser.add_argument("--values", type=float, nargs="+", help="Objective values.")
-        parser.add_argument("--state", type=str, default="complete", help="Trial state.")
+        parser.add_argument("--state", type=Optional[str], help="Trial state.")
```

Comment:
Arguments are optional by default if they're not explicitly defined with `required=True`.
```suggestion
        parser.add_argument("--state", type=str, help="Trial state.")
```